CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 14.0.0
+
+#### Breaking changes - [#525](https://github.com/openfisca/openfisca-core/pull/525)
+
+* Rename Variable attribute `url` to `reference`
+  - As previously, this attribute can be a string, or a list of strings.
+  - Variable attribute `law_reference` is removed.
```

Comment:
This should be a different item of the list, at the same level than "Rename...", and at the imperative mode.